services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/StatisticsToProtobufMapping.java

```diff
@@ -74,8 +74,8 @@ public Statistics constructProtobufStatistics() {
           .addAllKeyFigureCards(buildAllKeyFigureCards(jsonStringObjects))
           .build();
     } catch (BucketNotFoundException | ConnectionException | FilePathNotFoundException ex) {
-      logger.warn(ex.getMessage());
-      logger.warn("Statistics file will not be generated due to previous errors!");
+      logger.error(ex.getMessage());
+      logger.error("Statistics file will not be generated due to previous errors!");
```

Comment:
I would prefer to keep it in one single line... loggers behave from time to time differently and not all lines are kept always together - don't mentioning log analyzer tools...

```suggestion
      logger.error("{} - Statistics file will not be generated due to previous errors!", ex.getMessage());
```
or
```suggestion
      logger.error("Statistics file will not be generated due to previous errors!", ex);
```